packages/engine/src/map/MapUpdateSystem.ts

```diff
@@ -82,21 +86,21 @@ export default async function MapUpdateSystem(world: World): Promise<System> {
 
     if (mapState.needsUpdate) {
       // Perf hack: Start with an empty array so that any children that have been purged or that do not meet the criteria for adding are implicitly removed.
```

Comment:
is this comment still relevant after the change made below?

Reply:
Yeah, just decided (somewhat impulsively) to do it another way